server/playbook/playbook.go

```diff
@@ -72,6 +72,9 @@ type RequesterInfo struct {
 	UserID          string
 	TeamID          string
 	UserIDtoIsAdmin map[string]bool
+
+	// MemberOnly when true will return playbooks only if the requester is a member of those playbooks.
```

Comment:
nit: comment suggestion
```suggestion
 	// MemberOnly filters playbooks to those for which UserId is a member
```